ajax/libs/kindeditor/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "kindeditor",
+  "version": "4.1.11",
+  "filename": "kindeditor-all-min.js",
+  "description": "WYSIWYG HTML editor",
+  "keywords": [
+    "WYSIWYG",
+    "editor",
+    "HTML"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kindsoft/kindeditor.git"
+  },
+  "author": "Longhao Luo",
+  "license": "LGPL-2.1",
+  "homepage": "http://kindeditor.org",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kindsoft/kindeditor.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "kindeditor*",
+          "+(lang|themes)/**/*.+(js|css|gif|png)"
```

Comment:
@PeterDaveHello I remember that, we can't set partially minify action in one certain directory. But can we only set minify setting for `lang` and `themes`?

Reply:
I think it's fine.
There are some cases in CDNJS.
cc @extend1994 @PeterDaveHello

Reply:
We can't, what's the case? @sufuf3

Reply:
I answer the question `But can we only set minified setting for lang and themes?`
Sorry not mention detail.